presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2361,6 +2361,14 @@ public void testRightJoinWithNullValues()
     }
 
     @Test
+    public void testJoinWithDuplicateRelations()
+            throws Exception
+    {
+        assertQuery("SELECT * FROM orders JOIN orders USING (orderkey)");
+        assertQuery("SELECT * FROM lineitem x JOIN orders x USING (orderkey)");
```

Comment:
@martint 
Although I am able run these queries on Presto, these tests cause error in h2 parser.
```
testJoinWithDuplicateRelations(com.facebook.presto.tests.TestLocalQueries)  Time elapsed: 0.708 sec  <<< FAILURE!
org.skife.jdbi.v2.exceptions.UnableToCreateStatementException: org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ORDERS JOIN ORDERS USING ([*]ORDERKEY) "; SQL statement:
SELECT * FROM orders JOIN orders USING (orderkey) [42000-189] [statement:"SELECT * FROM orders JOIN orders USING (orderkey)", located:"SELECT * FROM orders JOIN orders USING (orderkey)", rewritten:"SELECT * FROM orders JOIN orders USING (orderkey)", arguments:{ positional:{}, named:{}, finder:[]}]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.getSyntaxError(DbException.java:191)
	at org.h2.command.Parser.getSyntaxError(Parser.java:517)
	at org.h2.command.Parser.prepareCommand(Parser.java:244)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at org.skife.jdbi.v2.DefaultStatementBuilder.create(DefaultStatementBuilder.java:52)
	at org.skife.jdbi.v2.SQLStatement.internalExecute(SQLStatement.java:1304)
	at org.skife.jdbi.v2.Query.fold(Query.java:173)
	at org.skife.jdbi.v2.Query.list(Query.java:82)
	at org.skife.jdbi.v2.Query.list(Query.java:75)
	at com.facebook.presto.tests.H2QueryRunner.execute(H2QueryRunner.java:125)
	at com.facebook.presto.tests.QueryAssertions.assertQuery(QueryAssertions.java:85)
	at com.facebook.presto.tests.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:94)
	at com.facebook.presto.tests.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:88)
	at com.facebook.presto.tests.AbstractTestQueries.testJoinWithDuplicateRelations(AbstractTestQueries.java:2367)
...
```